api/kyverno/v1/rule_types.go

```diff
@@ -64,7 +64,7 @@ type Rule struct {
 
 	// VerifyImages is used to verify image signatures and mutate them to add a digest
 	// +optional
-	VerifyImages []*ImageVerification `json:"verifyImages,omitempty" yaml:"verifyImages,omitempty"`
+	VerifyImages []ImageVerification `json:"verifyImages,omitempty" yaml:"verifyImages,omitempty"`
```

Comment:
This make some unit tests to fail...
Probably a sign that we are mutating something we shouldn't :thinking: 
cc @realshuting